content/docs/latest/deploying/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Deploying SPIRE
+short: Deploy
+kind: deploying
+aliases:
+    - /docs/latest/spire/using
+---
+
```

Comment:
For now, it would be great to have a couple of lines explaining to a reader the broad types of problems this content can help them solve.

Reply:
I added a summary - please review.